app/views/tag_wranglings/_wrangler_dashboard.html.erb

```diff
@@ -1,39 +1,62 @@
-<div id="dashboard" role="navigation region" class="tag wrangling region">
-  <ul class="navigation actions">
-    <% if current_user.is_a?(User) %>
-      <li><%= span_if_current(ts('Wrangling Home'), tag_wrangler_path(current_user)) %></li>
+<% if (logged_in_as_admin? && policy(:wrangling).full_access?) || current_user.try(:is_tag_wrangler?) || @counts %>
```

Comment:
:thinking: I thought this needed to be `read_access?` as well but the tests pass without it?

Reply:
It doesn't show on the tags index page, because that one doesn't have `@counts` set. So this should be changed and the tests should all start out in the tags index, not the tag wrangling tools